x-pack/plugins/siem/server/graphql/kpi_hosts/schema.test.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { graphql } from 'graphql';
+import { addMockFunctionsToSchema, makeExecutableSchema } from 'graphql-tools';
+
+import { rootSchema } from '../../../common/graphql/root/schema.gql';
+import { sharedSchema } from '../../../common/graphql/shared';
+import { Logger } from '../../utils/logger';
+import { ecsSchema } from '../ecs';
+import { dateSchema } from '../scalar_date';
+import { toBooleanSchema } from '../scalar_to_boolean_array';
+import { toDateSchema } from '../scalar_to_date_array';
+import { toNumberSchema } from '../scalar_to_number_array';
+import { sourceStatusSchema } from '../source_status/schema.gql';
+import { sourcesSchema } from '../sources/schema.gql';
+
+import { getKpiHostsQueryMock, mockKpiHostsData } from './kpi_hosts.mock';
+import { kpiHostsSchema } from './schema.gql';
+
+const testKpiHostsSource = {
+  id: 'Test case to query Siem Hosts KPIs data',
+  query: `
+    query GetOverviewHostsQuery(
+      $timerange: TimerangeInput!
+      $filterQuery: String
+    ) {
+      source(id: "default") {
+        KpiHosts(timerange: $timerange, filterQuery: $filterQuery) {
+          hosts
+          hostsHistogram {
+            key
+            count {
+              doc_count
+              value
+            }
+          }
+          authSuccess
+          authSuccessHistogram {
+            key
+            count {
+              doc_count
+              value
+            }
+          }
+          authFailure
+          authFailureHistogram {
+            key
+            count {
+              doc_count
+              value
+            }
+          }
+          uniqueSourceIps
+          uniqueSourceIpsHistogram {
+            key
+            count {
+              doc_count
+              value
+            }
+          }
+          uniqueDestinationIps
+          uniqueDestinationIpsHistogram {
+            key
+            count {
+              doc_count
+              value
+            }
+          }
+        }
+      }
+    }
+	`,
+  variables: {
+    timerange: {
+      interval: '12h',
+      to: 1514782800000,
+      from: 1546318799999,
+    },
+  },
+  context: {
+    req: {
+      payload: {
+        operationName: 'test',
+      },
+    },
+  },
+  expected: {
+    data: {
+      source: {
+        ...mockKpiHostsData,
+      },
+    },
+  },
+};
+
+describe('SIEM Hosts GQL Schema', () => {
```

Comment:
Since you added API Integration tests these `schema.test.ts` tests are no longer necessary -- feel free to remove this entire file :)